ubuntudesign/design.ubuntu.com
# Use assets from the assets server

I uploaded all images to the assets server (using [this script](https://gist.github.com/nottrobin/bad4d1b8f880bbb23ed10f5457d976bc), which I may make snap installable).

I then deleted all images from this repo, and updated all URLs in the markdown files.

QA
---

Run the site:

``` bash
bundle install
bundle exec jekyll serve
```

Check the whole site on <http://127.0.0.1:4000>, see that images all still work perfectly. Or at least as well as they ever did.
